portfolio/src/main/webapp/index.html

```diff
@@ -4,12 +4,14 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
     <script src="nav.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA2e6NZkMasOwEC1et7VznaQAdbU9NnlHE"></script>
```

Comment:
Just a note / no action needed: I think you're usually not supposed to commit private things like API keys to your Github repository, because that ends up making them publicly visible which means anyone can use them. In this case it doesn't really matter - the maps API is itself openly available - but it's something to remember in the future.

I actually don't know how you're supposed to be able to hide these keys and still deploy the app from this source.. it seems like it's probably possible, but not worth setting up here!